### PR TITLE
PPWM-54 add parameters to accept user project

### PR DESCRIPTION
**Pull request by dvoet**
https://broadworkbench.atlassian.net/browse/PPWM-54

**PR checklist**
- [x] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [x] Bump the version in `project/Settings.scala` `createVersion()`
- [x] Update `CHANGELOG.md` for those libraries
- [x] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [x] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [x] Get two thumbsworth of PR review
- [x] Verify all tests go green (CI _and_ coverage tests)
- [x] Squash commits and **merge to develop**
- [x] Delete branch after merge

### CODE REVIEW

**Review by jdcanas — APPROVED** [review]

File: google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleStorageInterpreter.scala [hunk 1]
```diff
@@ -393,57 +413,71 @@ private[google2] class GoogleStorageInterpreter[F[_]](
   override def setRequesterPays(bucketName: GcsBucketName,
                                 requesterPaysEnabled: Boolean,
                                 traceId: Option[TraceId] = None,
```
Comment (ehigham) [inline google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleStorageInterpreter.scala]: Is there a reason you're not defining / using the defaults from the trait here? (Im guessing that was the motivation behind removing the defaults in this type)

**Review by ehigham — APPROVED** [review]
Blobby.

File: google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleStorageInterpreter.scala (hunk 1 above)
Comment (dvoet) [reply google2/src/main/scala/org/broadinstitute/dsde/workbench/google2/GoogleStorageInterpreter.scala]: I just missed this one. And yes, the defaults were inconsistently used from the trait and duplicating them seemed silly so ...

**Review by Qi77Qi — APPROVED** [review]
this will be breaking change..so probably worth updating version

**Pull request merged**